onnxscript/tests/function_libs/torch_lib/ops_test.py

```diff
@@ -71,6 +71,27 @@ def _split_function_and_wrangler(
     return onnx_function_and_wrangler, None
 
 
+# according to https://pytorch.org/docs/stable/testing.html
+OPINFO_PRECISION_TABLE = {
+    # Relax atol and rtol for float32 based on empirical results
+    # The current most relaxed values are for aten::matmul
+    torch.float32: (3.7e-5, 1.8e-4),  # default is 1.3e-6, 1e-5
+    torch.float16: (1e-3, 1e-5),
+}
+
+
+def _get_rtol_atol_by_dtype(dtype: torch.dtype) -> tuple(Any, Any):
+    if dtype in OPINFO_PRECISION_TABLE:
+        return OPINFO_PRECISION_TABLE[dtype]
+    else:
+        return (0.0, 0.0)
+
+
+def _is_op_can_run_with_dtype(op_name: str, dtype: torch.dtype) -> bool:
```

Comment:
Suggest a new name: _dtype_is_supported_by_op()

Reply:
I found the previous implementation is return (None, None), so I followed it.